### ISSUE TITLE
Python Algorithm: How Do I Set a Value to Missing in a ~15minute Dataset?

### ISSUE THREAD

**Issue by JasonDSS** [opendcs/opendcs#437]
Greetings,

I'm trying to create a CCP Python algorithm to set a value to missing in a dataset with ~15minute interval when it falls outside of a range. I've tried the following 3, per suggestions from @MikeNeilson , but they don't seem to work. Anyone have any other suggestions? 

1)
globals()['revElevEast].value = Double.NEGATIVE_INIFINITY
algo.setOutput('revElevEast', Double.NEGATIVE_INFINITY)

2)
setMissing('revElevEast',Double.NEGATIVE_INFINITY)

3)
setOutput('revElevEast',Double.NEGATIVE_INFINITY)

Thanks!

Jason

**Comment by MikeNeilson** [comment]
I've reviewed the code. there's logic in the python handlers to prevent operations with missing data.
We should probably add a setMissing function, given that implementations differ in their missing value markers, to allow this behavior. 

Especially for CWMS Irregular and pseudo regular data these "valid database row of data that is missing but expected at this point in time" are rather important.

**Comment by ktarbet** [comment]
I started a branch in commit above.   Is this the direction you were thinking @MikeNeilson ?

**Comment by MikeNeilson** [comment]
Yes, but I missingValueMarker is going to be per implementation. Check NamedVariable itself. or perhaps the Screening algorithm has another missing value example in the java.
